src/Umbraco.Core/EmbeddedResources/BlockGrid/items.cshtml

```diff
@@ -17,9 +20,17 @@
             style=" --umb-block-grid--item-column-span: @item.ColumnSpan; --umb-block-grid--item-row-span: @item.RowSpan; ">
             @{
                 var partialViewName = "blockgrid/Components/" + item.Content.ContentType.Alias;
```

Comment:
This should now live inside the `else` below as we don't need it yet.

Reply:
@JasonElkin the variable is also used in the `catch` block :) 

```
@{
    var partialViewName = "blockgrid/Components/" + item.Content.ContentType.Alias;

    try
    {
        if (_selector.SelectComponent(item.Content.ContentType.Alias) is not null)
        {
            @await Component.InvokeAsync(item.Content.ContentType.Alias, item.Content)
        }
        else
        {
            @await Html.PartialAsync(partialViewName, item)
        }
    }
    catch (InvalidOperationException)
    {
        <p>
            <strong>Could not render component of type: @(item.Content.ContentType.Alias)</strong>
            <br/>
            This likely happened because the partial view <em>@partialViewName</em> could not be found.
        </p>
    }
}
```